api/Program.cs

```diff
@@ -1,30 +1,116 @@
 using api.Interfaces;
 using api.Services;
+using api.Models;
+using System;
+using System.Threading.Tasks;
+using System.Configuration;
+using System.Collections.Generic;
+using System.Net;
+using Microsoft.Azure.Cosmos;
 
-var builder = WebApplication.CreateBuilder(args);
+public class Program
+{
+    private static readonly string EndpointUri = "https://kemcowiki.documents.azure.com:443/";
+    private static readonly string PrimaryReadOnlyKey = "sJ8scLbCX0hi5yswpqM1cMjbGQK1Yasg5f4aWVV1TtNza2c22x8CrYsPFo0x20aAqLxucLBZyuG3cF3kMBjN1A==";
+    private CosmosClient cosmosClient;
+    private Database database;
+    private Container container;
+    private string databaseId = "kemcogames";
+    private string containerId = "container1";
+    
+    public static async Task Main(string[] args)
+    {
+        var builder = WebApplication.CreateBuilder(args);
 
-// Add services to the container.
+        builder.Services.AddControllers();
+        // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+        builder.Services.AddEndpointsApiExplorer();
+        builder.Services.AddSwaggerGen();
 
-builder.Services.AddControllers();
-// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
-builder.Services.AddEndpointsApiExplorer();
-builder.Services.AddSwaggerGen();
+        builder.Services.AddScoped<IGameService, GameService>();
 
-builder.Services.AddScoped<IGameService, GameService>();
+        var app = builder.Build();
 
-var app = builder.Build();
+        // Configure the HTTP request pipeline.
+        if (app.Environment.IsDevelopment())
+        {
+            app.UseSwagger();
+            app.UseSwaggerUI();
+        }
 
-// Configure the HTTP request pipeline.
-if (app.Environment.IsDevelopment())
-{
-    app.UseSwagger();
-    app.UseSwaggerUI();
-}
+        app.UseHttpsRedirection();
+
+        app.UseAuthorization();
 
-app.UseHttpsRedirection();
+        app.MapControllers();
+        //////// CosmosDB test
+        try
+        {
+            Console.WriteLine("Beginning Azure CosmosDB access test...\n");
+            Program p = new Program();
+            await p.GetStartedDemoAsync();
+        }
+        catch (CosmosException de)
+        {
+            Exception baseException = de.GetBaseException();
+            Console.WriteLine("{0} error occurred: {1}", de.StatusCode, de);
+        }
+        catch (Exception e)
+        {
+            Console.WriteLine("Error: {0}", e);
+        }
+        finally
+        {
+            Console.WriteLine("End of demo, press any key to exit.");
+            Console.ReadKey();
+        }
+        /////////// Now run API service
+        app.Run();
+    }
 
-app.UseAuthorization();
+    public async Task GetStartedDemoAsync()

```

Comment:
Long term we'll slip this into a DB class but if everything works here then the DB should be good to go

Reply:
db is good to go but def should be moved out